src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -832,6 +832,7 @@ sylius_review:
 sylius_grid:
     templates:
         action:
+            create_zone: SyliusAdminBundle:Zone/Action:createZone.html.twig
```

Comment:
Yeah, but all other actions are here too right now, we will move them later. Good catch!